product/models.py

```diff
@@ -2,6 +2,9 @@
 from django.core.validators import MinValueValidator, MaxValueValidator
 
 class Product(models.Model):
+    ko_name   = models.CharField(max_length=128)
+    en_name  = models.CharField(max_length=128)
```

Comment:
변수명 줄이지 말아주세요. 위코드 컨벤션입니다.
= 기준으로 정렬하는 것도 잊지 말아주세요.
```suggestion
    korean_name   = models.CharField(max_length=128)
    english_name  = models.CharField(max_length=128)
```

Reply:
다음부터 라벨 꼭 달겠습니다!